charts/jenkins/templates/jenkins-backup-cronjob.yaml

```diff
@@ -29,6 +29,15 @@ spec:
         spec:
           restartPolicy: OnFailure
           serviceAccountName: {{ template "jenkins.fullname" . }}-backup
+{{- if .Values.backup.usePodSecurityContext }}
+          securityContext:
+            runAsUser: {{ default 0 .Values.backup.runAsUser }}
+{{- if and (.Values.backup.runAsUser) (.Values.backup.fsGroup) }}
+{{- if not (eq (int .Values.backup.runAsUser) 0) }}
+            fsGroup: {{ .Values.backup.fsGroup }}
+{{- end }}
+{{- end }}
+{{- end }}
```

Comment:
Would be nice if you could indent those if statement.

Reply:
sure @torstenwalter, indented the if statements